### ISSUE TITLE
Add a value field to event tags

### ISSUE THREAD

**Issue by iglocska** [MISP/MISP#1261]
Right now tags are hard coded, but it would make sense to have taxonomy tags with a variable component. 

For example:

namespace:tag=$value

where $value is a variable component stored on the instance of the tag usage (so in the event tag object)

**Comment by truckydev** [comment]
Very good idea, it could also be interesting to do the same for attributes.

Maybe not use the same name "taxonomy" field to avoid confusion with the Event. 

But use an equivalent function for attributes would be really helpful for the relevance of attributes in an Event. 
attrNamespace:tag=$value

**Comment by iglocska** [comment]
Indeed, we plan on allowing the option to tag attributes the same way as we tag events in the near future. :)
